paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -101,18 +104,38 @@ class FCPrimitiveFactory {
     return dst_mem;
   }
 
+  mkldnn::memory Reorder(const memory& src_mem,
```

Comment:
What's the function of the `Reorder`? Please add some comments here.

Reply:
Reorder transforms mkl-dnn memory from one data format to another. More data can be found [here](https://intel.github.io/mkl-dnn/understanding_memory_formats.html).

In the particular overload you have mentioned, it also scales the resulting data. The particular use case of this function is to quantize weights.